extensions/resteasy/deployment/pom.xml

```diff
@@ -38,6 +38,10 @@
             <groupId>com.oracle.substratevm</groupId>
             <artifactId>svm</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-security-deployment</artifactId>
```

Comment:
RESTeasy should not have a hard dependency on the security extension. I think the exceptions should probably be moved to the quarkus-security project to avoid the runtime dependency, and just copy the trivial utility method used in the -deployment module.

Reply:
done, now it's only a test dependency

Reply:
done